MagnetoSoftwareTesting/src/test/java/MagnetoTest/MagnetoApplicationTest.java

```diff
@@ -0,0 +1,257 @@
+		log.info("enter street address");
+		driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(50));
+		By cityBy = By.xpath("//input[@name='city']");
+		wait.until(ExpectedConditions.visibilityOfElementLocated(cityBy)).sendKeys(city);
+		wait.until(ExpectedConditions.visibilityOfElementLocated(emailBy)).clear();
+		log.info("enter city");
+		driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(50));
+		By postcodeBy = By.xpath("//input[@name='postcode']");
+		wait.until(ExpectedConditions.visibilityOfElementLocated(postcodeBy)).sendKeys(postal_code);
+		wait.until(ExpectedConditions.visibilityOfElementLocated(emailBy)).clear();
+		log.info("enter postcode");
+		driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(50));
+		By countryBy = By.xpath("//select[@name='country_id']//option[@data-title='India']");
+		wait.until(ExpectedConditions.visibilityOfElementLocated(countryBy)).click();
+		log.info("click on country");
+		driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(50));
+		By regionBy = By.xpath("//select[@name='region_id']//option[@data-title='Andhra Pradesh']");
+		wait.until(ExpectedConditions.visibilityOfElementLocated(regionBy)).click();
+		log.info("click on state");
+		driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(50));
+		By phoneBy = By.xpath("//input[@name='telephone']");
+		wait.until(ExpectedConditions.visibilityOfElementLocated(phoneBy)).sendKeys(phone);
+		wait.until(ExpectedConditions.visibilityOfElementLocated(emailBy)).clear();
+		log.info("enter mobile number");
+		driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(50));
+		By keymethodBy = By.xpath("//*[@id='label_carrier_flatrate_flatrate']");
+		wait.until(ExpectedConditions.visibilityOfElementLocated(keymethodBy)).click();
+		log.info("click on shipping methods");
+		driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(50));
+		By nextbuttonBy = By.xpath("//button[@class='button action continue primary']");
+		wait.until(ExpectedConditions.visibilityOfElementLocated(nextbuttonBy)).click();
+		log.info("click on next button");
+		driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(50));
+	}
+
+	/**
+	 * @param Verify                         order summary
+	 * @param product_name
+	 * @param product_quantity,product_color ,product_size and product_price
+	 * @return
+	 */
+	@Test(priority = 5)
+	public void VerifyOrderSummary() {
+		WebElement ordersummary = driver.findElement(By.xpath("//div[@class='opc-block-summary']"));
+		wait.until(ExpectedConditions.visibilityOf(ordersummary));
+		driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(50));
+		By productdetailBy = By.xpath("//div[@class='block items-in-cart']/div");
+		wait.until(ExpectedConditions.visibilityOfElementLocated(productdetailBy));
+		WebElement product = driver.findElement(productdetailBy);
+		JavascriptExecutor jse = (JavascriptExecutor) driver;
```

Comment:
Create pagebase and add common JS click method there and reuse wherever you want